### DISCUSSION TITLE
Speed settings

### DISCUSSION THREAD

**Discussion by 0709wiwiwi** [devemux86/cruiser#233] [Q&A] [answered]
Routes > Speeds > Route duration > Average speed.

_Each vehicle type has a corresponding avg speed setting.
Setting the average speed is clear and does an excellent job.
But what is the purpose of the min and max speed setting?_

**Comment by devemux86** [comment, marked as answer]
@0709wiwiwi the selected min / max speeds are used in speed auto zoom, please see here:
- https://github.com/devemux86/cruiser/discussions/63

**Comment by 0709wiwiwi** [comment]
OK, Roger ,Thanks.
